### DISCUSSION TITLE
Host dashboard dependency in a custom server

### DISCUSSION THREAD

**Discussion by Hunsu** [renovatebot/renovate#12072] [Suggest an Idea]
I used renovate with Gitlab and I really like the dashboard page. Now my company uses a self hosted bitbucket and the dashboard isn't available for that platform.

I'm thinking of forking this project and make an app that host the dashboard.

I am wondering if a such a feature interests you and if so I would love to discuss the implementation with you.

**Comment by rarkins** [comment]
Such a feature would not be a good fit for this main repo as it's a stateless CLI tool and I don't want to mis stateful things like a web server or categorizing logs. I am planning a full refresh of the current app dashboard (which is pretty basic to be honest..) and will keep in consideration the option of making it self-hostable, but that's probably a "1st half 2022" thing so I wouldn't advise you to stop..

**Reply by Hunsu** [reply]
Thank you for your response. Just for clarification I wasn't thinking of putting the server on this repo.

Also thanks for considering the self host of dashboard page.

**Reply by qcn** [reply]
@rarkins I'd love a self-hosted dashboard option - our use-case is that issues are disabled in our Github repo (because we centralise them elsewhere), but we'd still like to be able to use the Renovate dashboard :)

**Comment by Hunsu** [comment]
@qcn 
I did a poc for my company. Here are the links to my fork of renovate and a simple self-hosted dashboard (nothing fancy)
https://github.com/Hunsu/renovate-simple-dashboard
https://github.com/Hunsu/renovate/commit/186237b21113722709e2d8a745fe68194491e23e

**Reply by qcn** [reply]
Thank you! Do you know if it's possible for your dashboard to work with the WhiteSource Github app, or only for self-hosted Renovate?

**Reply by Hunsu** [reply]
No you need my fork of renovate to be able to use the self hosted dashboard.
